e2e-test/src/main/java/org/alfresco/hxi_connector/e2e_test/CreateNodeTest.java

```diff
@@ -0,0 +1,169 @@
+/*-
+ * #%L
+ * Alfresco HX Insight Connector
+ * %%
+ * Copyright (C) 2023 - 2024 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.hxi_connector.e2e_test;
+
+import static io.restassured.RestAssured.given;
+
+import java.io.File;
+
+import io.restassured.response.Response;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.containers.localstack.LocalStackContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.wiremock.integrations.testcontainers.WireMockContainer;
+
+import org.alfresco.hxi_connector.common.test.util.DockerContainers;
+
+@Slf4j
+@Testcontainers
+public class CreateNodeTest
```

Comment:
Class name should probably contain `IntegrationTest` to avoid running it during `mvn package` lifecycle. We might also consider adding new `E2ETest` suffix 🤔

Reply:
Those tests should differ from integration tests and current PR is more about working testcontainers so maybe we can add E2ETest suffix in separate PR

Reply:
So lets add for now the `IntegrationTest` suffix and fix it in next PR. Ale question is: when this tests should run - at which stage or in which phase?

Reply:
Next question is: if they should be located in `src/main/` dir and packed into to `jar` file?

Reply:
Name of the test and test class changed to contain IntegrationTest suffix and was moved to src/test as advised